Frameworks/include/CAAnimationInternal.h

```diff
@@ -29,22 +30,21 @@ struct CAMediaTimingProperties {
 @interface CAAnimation () {
 @public
     CAMediaTimingProperties _timingProperties;
-    DisplayAnimation *_runningAnimation, *_runningAnimation2;
+    std::shared_ptr<ILayerAnimation> _runningAnimation;
     id _attachedLayer;
     idretaintype(NSString) _keyName;
 }
 
 - (void)setDelay:(double)delay;
-- (void)_abortAnimation;
 - (void)_removeAnimationsFromLayer;
 - (void)_setFinishedSelector:(SEL)selector;
 @end
 
 @interface NSObject (CAAnimationInternal)
-- (void)_removeAnimationsFromLayer;
 - (BOOL)wasRemoved;
 - (BOOL)wasAborted;
```

Comment:
>(BOOL)wasAborted; [](start = 2, length = 17)

Do we still need this "internal" wasAborted property if we removed the internal _abortAnimation method?